### DISCUSSION TITLE
2023-11-16 THE END

### DISCUSSION THREAD

**Discussion by hanseulhee** [10-rolling/rollingPolling#103] [Daily_Meeting]
### ✅ 어제 해낸 일

- [x] list page 반응형
- [x] 페이지, 컴포넌트 반응형 추가
- [x] SendMessage 페이지에서 이미지 위치 변경
- [x] API 공통 에러 처리 기능 구현
- [x] 개발 마무리

### 📝 회의내용

**발표 준비하기**

### 💪🏻 오늘 할일

우리의 마지막을 .... 즐기기 🥲🤍
쫑파티 🔥🔥🔥🔥

### 🤔 특이사항

안녕은 영원한 헤어짐은 아니겠지요 ~~~~ 🥺
10팀 잊지 않겠습니다 .. 😢

### 발표자료

[10팀.pdf](https://github.com/10-rolling/rollingPolling/files/13389105/10_.2.pdf)

**Comment by miniposi** [comment]
안돼 ㅠㅠ 헤어지지 말아요... 🥲

**Reply by hanseulhee** [reply]
울지마세요 수님!! '수'리스리마스리

**Comment by ngsy** [comment]
10점만점의 10팀!🤓
